apps/readest-app/src/types/system.ts

```diff
@@ -26,6 +26,11 @@ export interface AppService {
   hasWindowBar: boolean;
   isAppDataSandbox: boolean;
 
+  listen?(
```

Comment:
The name “listen” feels very Tauri-specific. Can we consider a different name that is more generic? How about `subscribe`? And even other platforms have no implementation of this function they should have a no-op definition w or w/o a warning log to keep the interface consistent.